client/src/components/ForgotPassword/style.scss

```diff
@@ -0,0 +1,84 @@
+.login-container {
+  .forgot-password-wrapper {
+    width: 100%;
+    max-width: 400px;
+    margin: 200px 0;
+
+    ::placeholder {
```

Comment:
I believe we have these styles are in general file

Reply:
fixed